ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:06:00
-#PBS -l select=1:mpiprocs=48:ompthreads=1:ncpus=48:mem=24GB
+#PBS -l select=1:mpiprocs=128:ompthreads=1:ncpus=128:mem=28GB
```

Comment:
Please revert this change or justify why 128 cores are being requested.
The memory increase is acceptable.
@KateFriedman-NOAA will need to update `config.resources.*`

Reply:
Jan 21, 2022 a merge in:https://github.com/lgannoaa/global-workflow/commit/45dad23d728226391e8be752f424097acc84a67a
 has changed it from:
---
#PBS -l place=vscatter,select=1:mpiprocs=128:ompthreads=1:ncpus=128
#PBS -l place=excl
---
to
---
#PBS -l place=vscatter,select=1:mpiprocs=48:ompthreads=1:ncpus=48:mem=24GB
---
Therefore, it is save to revert it back.